src/Analyzers/Analyzers.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsRoslynComponent>true</IsRoslynComponent>
+
+    <!-- Do not include the generator as a lib dependency -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <IncludeShared>false</IncludeShared>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.DurableTask.Analyzers</PackageId>
```

Comment:
This can be removed - `PackageId` defaults to `AssemblyName`, and assembly name will be set to `Microsoft.DurableTask.{ProjectName}`

https://github.com/microsoft/durabletask-dotnet/blob/main/Directory.Build.props#L13

Reply:
Ah ok, fixed, thanks for the explanation